examples/config/roger-mesos-tools.config

```diff
@@ -5,7 +5,7 @@ default_username: some_user_name
 environments:
   local:
     chronos_endpoint: http://localmesos01:4400
-    haproxy_config_path: :8000/config
+    proxy_config_path: :8000/config
```

Comment:
Does this need to get changed to `:8000/api/state` ?

Reply:
Yep, might as well keep the example updated. e0c167caf7a49d60696dab0218e5e18bbdbc78ef